js/apps/system/_admin/aardvark/APP/react/src/views/query/useSavedQueriesHandlers.tsx

```diff
@@ -123,6 +123,8 @@ const patchQueries = async ({
   onSuccess: () => void;
 }) => {
   const storageKey = getQueryStorageKey();
+  const currentUser = window.App.currentUser || "root";
+
```

Comment:
This works. You could also think about solving this like this:
```suggestion
  const currentUser = window.App.currentUser;
  if (!frontendConfig.authenticationEnabled) {
    currentUser = "root";
  }
```

To make it a little bit "more clear" why we're setting it to root here.

Reply:
Great feedback! Done!